examples/tracetest-opensearch/docker-compose.yml

```diff
@@ -0,0 +1,82 @@
+version: '3'
+services:
+
+    tracetest:
+        container_name: tracetest
+        image: kubeshop/tracetest:v0.6.4
+        volumes:
+            - type: bind
+              source: ./tracetest-config.yaml
+              target: /app/config.yaml
+        ports:
+            - 8080:8080
+        depends_on:
+            postgres:
+                condition: service_healthy
+            otel-collector:
+                condition: service_started
+
+    postgres:
+        container_name: postgres
+        image: postgres:14
+        environment:
+            POSTGRES_PASSWORD: postgres
+            POSTGRES_USER: postgres
+        ports:
+            - 5432:5432
+        healthcheck:
+            test: pg_isready -U "$$POSTGRES_USER" -d "$$POSTGRES_DB"
+            interval: 1s
+            timeout: 5s
+            retries: 60
+
+    otel-collector:
+        image: otel/opentelemetry-collector:0.54.0
+        ports:
+            - "55679:55679"
+            - "4317:4317"
+            - "8888:8888"
+        command:
+            - "--config"
+            - "/otel-local-config.yaml"
+        volumes:
+            - ./collector.config.yaml:/otel-local-config.yaml
+        depends_on:
+            - data-prepper
+
+    data-prepper:
+        restart: unless-stopped
+        container_name: data-prepper
```

Comment:
This one is required, for some reason, for the connections to work. Sounds like a bug, and I'd look into removing this setting, because it can conflict with existing containers

Reply:
Are you sure? It worked fine on my machine after I removed the container_name from all services

Reply:
Yea, I `docker compose down` and `up` without the names and it worked now. Probably had some leftover config